packages/shared/styles/components/atoms/SfInput.scss

```diff
@@ -22,14 +22,14 @@
     background: var(--input-label-background);
     transform: var(
       --input-label-transform,
-      translate3d(0, calc(-1 * var(--input-label-top, 50%)), 0)
```

Comment:
question: isn't this a breaking change, as if someone overrides this ```var(--input-label-top)``` in its project then this will affect the translation?

Reply:
Yes, it is. I will describe it in migration guide.